tests/agent/actions/test_draft_datasource_config.py

```diff
@@ -170,7 +170,9 @@ def test_test_draft_datasource_config_success_sql_ds(
     assert action_result.created_resources == []
 
     # assert that the action properly calls helper methods to get table names and update the draft config
-    _get_table_names_spy.assert_called_with(datasource=datasource_cls(**datasource_config))
+    _get_table_names_spy.assert_called_with(
+        datasource=datasource_cls(**datasource_config), include_views=True
+    )
     _update_table_names_list_spy.assert_called_with(config_id=config_id, table_names=table_names)
```

Comment:
These assertions don't add any value to the test and make refactoring harder. Testing that a method calls a private helper makes no guarantee that what the helper is doing is correct.

These tests is already verifying that the correct endpoint is called with the correct tables/views and that is enough

Reply:
Some tests here should guarantee that we return views as well